core/src/main/com/sinch/sdk/core/http/AuthManager.java

```diff
@@ -13,6 +14,11 @@ public interface AuthManager {
 
   void resetToken();
 
+  default Collection<Pair<String, String>> getAuthorizationHeaders(
+      String method, String httpContentType, String path, String body) {
+    return getAuthorizationHeaders(Instant.now().toString(), method, httpContentType, path, body);
```

Comment:
Why do you define the timestamp at this level?
The other parameters are static, coming from the specification. This one is dynamic at could be generated in the `VerificationApplicationAuthManager` as it was before.